### DISCUSSION TITLE
CO2 emissions for hydro storage

### DISCUSSION THREAD

**Discussion by simaker** [electricitymaps/electricitymaps-contrib#4774] [Q&A] [answered]
Why are there emissions for hydro storage? The electricity is already produced and only stored. So except for losses there are not additional emissions. If the average emisions are taken, they will be double in the whole sum of emissions.

**Comment by thomasgibon** [comment, marked as answer]
Thanks for opening the discussion, from what I understand, there are different ways to account for hydro storage emissions.

1. Exclude electricity generation from pumped hydro, since it's only postponing the final use of electricity produced elsewhere. This should also be done for all storage equipment for consistency. Pros: no double-counting, Cons: during pumped hydro discharge, load will be associated with emission-free electricity.
2. Include electricity generation from pumped hydro, and exclude electricity used for the pumping (**BUT** keep track of its carbon intensity). Pros: no double-counting and more precise time-wise, Cons: difficult to implement, you need a new time series of "carbon intensity of consumption during pumping for country X" and of the corresponding volume to exclude.

From what I know, solution 1 is used by the IEA, solution 2 is used by the ecoinvent database. I think it's a matter of what is technically feasible.

EDIT: ElectricityMaps is actually using solution 2, see the nice post here: https://github.com/electricitymaps/electricitymaps-contrib/wiki/Carbon-intensity-calculations#storage

**Comment by christopheradlam** [comment]
@thomasgibon has covered the methodologies currently employed quite well. 

I'll just add that the reason PHES has emissions is the same reason wind, solar and regular hydro have emissions: construction and decommissioning (and methane from flooding could also be included), which are not emissions-free processes and these must be included in the lifecycle emissions for the technology.
